source/sass/main.scss

```diff
@@ -20,6 +20,7 @@ html,
 body {
   margin: 0;
   padding: 0;
+  scroll-behavior: smooth;
```

Comment:
Yeah, not in chrome in any way that's meaningful.

Hilariously enough, MDN showed this:

"Basic support Chrome (Yes)[1]"

But if I scroll down:

"[1] Supported in Chrome and Opera by enabling the "Smooth Scrolling" or "Enable experimental web platform features" flag."

See: https://developer.mozilla.org/en/docs/Web/CSS/scroll-behavior

My guess is chrome gets it "soon"